examples/embed/embed_multiple.py

```diff
@@ -77,6 +77,6 @@
                        div=div)
 
 with open(filename, 'w') as f:
-    f.write(html)
+    f.write(html.encode('utf8'))
```

Comment:
A better solution would be to use `io.open(..., encoding='utf-8')`, especially using it uniformly across code base.